cfme/automate/provisioning_dialogs.py

```diff
@@ -112,34 +67,77 @@ def type_nav(self):
         return self.type[0]
```

Comment:
This may change totally as the nav name is only to do with force_navigate, I was going to do a secondary review of it once we had done the conversion.